Header header
float32 rssi
int16   column
int16 buffer_size
float32 mean
float32 median
float32 dt
float32 hz
int16 peak_column
float32 peak_value
float32 peak_frequency
float32 lower_freq_mhz
float32 upper_freq_mhz
float32[] data

